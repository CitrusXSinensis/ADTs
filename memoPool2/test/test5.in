ra1 1 1 1 1 1 1 1 1 1 1x
pa
ca
aa 1
aa 1
aa 1
aa 1
aa 1
pa
ca
aa 1
pa
ca
fa 2
fa 3
pa
ca
fa 1
pa
ca
