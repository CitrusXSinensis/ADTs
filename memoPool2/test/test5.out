1 1 1 1 1 1 1 1 1 1 1
11/16
1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
16/16
1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
17/32
Nullptr
Nullptr
1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
17/32
Chunk: 1
1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
16/32
